src/librustc_mir/transform/preallocate_gc_contents.rs

```diff
@@ -0,0 +1,77 @@
+#![allow(dead_code)]
```

Comment:
Kill these before merge?

Reply:
These or this?

Reply:
dead code and unused imports.

Reply:
Can you be clear to Jake which line numbers you're talking about? Otherwise it's a bit hard for him to know exactly what you're suggesting.

Reply:
> dead code and unused imports.

Lines 1 and 2 respectively.

Reply:
Good catch. This is debugging code that I had left in.